src/Snippets/README.md

```diff
@@ -25,8 +25,13 @@ public static void MySnippet()
 }
 ```
 
-In your Markdown documentation, refer to your code snippet by adding `<!-- snippet: my-snippet -->` and `<!-- endSnippet -->` comments to your Markdown file.
+In your Markdown documentation, refer to your code snippet by adding `snippet: my-snippet`.
```

Comment:
We should keep this note unchanged or just improve it. The wrapping in xml comments in required.

Example markdown content:

``` text
<!-- snippet: synchronization-context -->
```cs
// Retrieve an instance of ResilienceContext from the pool
// with the ContinueOnCapturedContext property set to true
ResilienceContext context = ResilienceContextPool.Shared.Get(continueOnCapturedContext: true);

await pipeline.ExecuteAsync(
    static async context =>
    {
        // Execute your code, honoring the ContinueOnCapturedContext setting
        await MyMethodAsync(context.CancellationToken).ConfigureAwait(context.ContinueOnCapturedContext);
    },
    context);

// Optionally, return the ResilienceContext instance back to the pool
// to minimize allocations and enhance performance
ResilienceContextPool.Shared.Return(context);
```
<!-- endSnippet -->
```

Reply:
You don't need to use the _lengthy_ xml comments. The shorthand code will generate the exact same output.

https://github.com/App-vNext/Polly/assets/57183693/b15d83ca-edac-4c97-9014-d785dcb599ec

Reply:
Oh nice :)